backend-project/small_eod/letters/admin.py

```diff
@@ -47,7 +46,7 @@ def download_selected_letters(modeladmin, request, queryset):
                 r = requests.get(file_path, stream=True)
 
                 z.write_iter(
-                    f"{case_id}-{case_name}/{ordering}-{filename}",
+                    f"{case_id}-{case_name}/{filename}",
```

Comment:
Ja proponuje jednak zachować tu numeracje dla plików, bo po wypakowaniu łatwiej się plikami posługiwać, bo zachowują kolejność.  Stosowałbym także dwucyfrową numeracje.

Reply:
Ja myślę, że numeracja powinna być globalna dla wszystkich plików z sprawy. Listy sortowane według daty odbioru/nadania, a później załączniki dla każdego listu.

To znaczy:
- 01. załącznik nr 1 listu nr 1
- 02. załącznik nr 2 listu nr 1
- 03. załącznik nr 1 listu nr 2

Nie rozumiem pytania o ścieżkę. W tym przypadku nie dodajesz żadnego letter, a pobierasz istniejące dane. Ścieżka dotychczasowa była poprawna, ale skasowałeś `ordering` nie zastępując go niczym, co uważam za nie optymalne.

Reply:
Kwestie numeracji rozumiem. 
W pytaniu o ścieżkę, chodziło mi w jaki sposób dodać plik do letter żeby później przetestować działanie wprowadzonych zmian (czyli pobrać załączone pliki)? 
<img width="1128" alt="Zrzut ekranu 2020-10-1 o 23 07 57" src="https://user-images.githubusercontent.com/9075439/94863598-f9c53f80-043a-11eb-8a3b-018563240542.png">

Reply:
Teraz rozumiem. Faktycznie w przypadku Django Admina nie jest to przystępne, bo planujemy bardziej rozbudowaną logikę po stronie klienta. Pliki są składowane w Minio. Zobacz na pełne flow na https://github.com/watchdogpolska/small_eod/blob/dev/backend-project/small_eod/letters/tests/test_views.py#L39-L63
Plik należy umieścić w buckecie w Minio i podać ścieżkę względną wobec bucketu np. `8a1a222e-03e7-442f-9063-41cabf42b16f/text.file` dla:
```bash
$ ls -lah ./dev-data/minio/files/8a1a222e-03e7-442f-9063-41cabf42b16f/text.file 
-rw-r--r-- 1 root root 3 paź  1 23:21 ./dev-data/minio/files/8a1a222e-03e7-442f-9063-41cabf42b16f/text.file
```

Reply:
Poprawione :)

Reply:
Miałem na myśli globalny licznik (1,2,3), a u Ciebie jest (0,1,0), ale proponuje tego nie zmieniać, a poczekać na uwagi użytkowników, bo mogą mieć różne potrzeby w praktyce.

Reply:
Problem jaki miałem z minio to w Letters podawałem ścieżkę: http://127.0.0.1:9000, która działała mi z przeglądarki... a api komunikuje się z minio na adresie 1272.18.0.4:9000. - info dla potomnych. 
Skoro może zostać tak jak jest i został wykonany merge, to czy mogę usunąć brancha i dostać pozytywne review do PR czy dalsze uwagi będą zgłaszane w ramach tego PR i zostawić ten branch  ?

Reply:
Możesz usunąć brancha. Dla PR, który został zmergowany nie ma możliwości niestety wystawienia pozytywnego review. Jest on domyślny z tytułu wcielenia zmian. Ja nie mam dalszych uwag, a miałem tylko komentarz, że zostawiam jak jest i zobaczymy uwagi prawdziwych użytkowników.

Reply:
Szkoda, bez review nie zaliczył mi się PR w Hacktoberfest :( 
jak byś coś dla mnie miał to wołaj :)

Reply:
> Szkoda, bez review nie zaliczył mi się PR w Hacktoberfest :(

Zaliczy :) W tym repozytorium liczy mi nawet jak sam zrobię PR i sam go zmerguje, a sam nie mogę sobie zrobić review.

> > Do pull requests have to be accepted/merged?
> No. They will still count unless they are spam, irrelevant, or labelled as invalid or spam by a maintainer, as long as they are marked as ready-for-review. (Draft pull requests are not counted.)